### PR TITLE
feat(table): add column title attribute in expandable props.

### PR DESCRIPTION
**Pull request by linaaaqi**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
#36490 

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | add `column title` attribute in `expandable` props. |
| 🇨🇳 Chinese | `expandable` 属性中增加 `column title` 配置 |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
feature branch

**Comment by losgif** [comment]
> feature branch

改了

**Comment by afc163** [comment]
冲突了

File: components/table/index.zh-CN.md [hunk 1]
```diff
@@ -178,6 +178,7 @@ const columns = [
 | 参数 | 说明 | 类型 | 默认值 | 版本 |
 | --- | --- | --- | --- | --- |
 | childrenColumnName | 指定树形结构的列名 | string | children |  |
+| columnTitle | 自定义展开列表头 | ReactNode | - | 4.23.0 |
```
Comment (afc163) [inline components/table/index.zh-CN.md:181]: 英文文档。

Comment (losgif) [reply components/table/index.zh-CN.md:181]: 加了

**Pull request closed without merging**